### PR TITLE
Rename wrong doc-string in observer stub

### PR DESCRIPTION
**Pull request by fathur**
This PR intended to fix the wrong doc-string when generating observer for specific model

### CODE REVIEW

**Review by girardinsamuel — APPROVED** [review]
Thanks for finding this out and fixing this ! Looks good to me 👍

**Comment by fathur** [comment]
Thanks, @girardinsamuel. This is my first-time contribution, it seems needs maintainer @josephmancuso approval also

**Review by josephmancuso — APPROVED** [review]

**Pull request merged**
